packages/legend-extension-dsl-serializer/src/models/metamodels/pure/model/packageableElements/store/Binding.ts

```diff
@@ -44,15 +54,15 @@ export class Binding extends Store implements Hashable {
     });
   }
 
-  setSchemaSet(value: string): void {
+  setSchemaSet(value: OptionalPackageableElementReference<SchemaSet>): void {
```

Comment:
We should do this instead `setSchemaSet(value: SchemaSet | undefined)`, see `PureInstanceSetImplementation.setSrcClass` for example